parm/config/gfs/config.ufs

```diff
@@ -126,20 +136,23 @@ case "${fv3_res}" in
           export NEST_DLON=0.25
           export NEST_DLAT=0.25
           export psl_gwd_dx_factor=6.0
-          export WRITE_GROUP_GDAS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=2
-          export WRITE_GROUP_GFS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=2
         else
+          if [[ "${RUN}" = *gdas ]]; then
+            export nthreads_fv3=1
+            export nthreads_ufs=1
+            export layout_x=2
+            export layout_y=2
+            export WRITE_GROUP=1
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
+          elif [[ "${RUN}" = *gfs ]]; then
+            export nthreads_fv3=1
+            export nthreads_ufs=1
+            export layout_x=2
+            export layout_y=2
+            export WRITE_GROUP=1
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
+          fi
```

Comment:
Since these are the same, there's no need for an if-elif block:
```suggestion
            export nthreads_fv3=1
            export nthreads_ufs=1
            export layout_x=2
            export layout_y=2
            export WRITE_GROUP=1
            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
```